README.md

```diff
@@ -18,9 +18,9 @@ Repo contains the steps to re-create an application running on Heroku server (al
     * [Create 2 apps (UAT/Prod)](heroku.md#create-2-apps-uatprod)
     * [Create a pipeline](heroku.md#addattach-apps-to-pipeline)
     * [Add/Attach apps to pipeline](heroku.md#addattach-apps-to-pipeline)
+    * [Add Buildpack](heroku.md#add-buildpack)
     * [Configure addons (explain)](heroku.md#configure-addons-explain)
     * [Add ENV variables](heroku.md#add-env-variables)
-    * [Add Procfile](heroku.md#add-procfile)
```

Comment:
@pro-pooja : Procfile link was removed, is it by mistake or intentionally removed?

Reply:
@sssurii , We don't need to add it separately, it is already added in the boilerplate.

Reply:
@pro-pooja : You are right when we are deploying boilerplate on Heroku. Think about when we are deploying any other app (not using boilerplate) so Procfile becomes an essential part of knowledge when talking about deploying on Heroku. 
So we should still keep it. 
@svikramjeet what do you think about it?